Server/mods/deathmatch/logic/packets/CPlayerPuresyncPacket.cpp

```diff
@@ -36,8 +36,8 @@ bool CPlayerPuresyncPacket::Read(NetBitStreamInterface& BitStream)
             return false;
 
         // Only read this packet if it matches the current time context that
-        // player is in.
-        if (!pSourcePlayer->CanUpdateSync(ucTimeContext))
+        // player is in. Allow position updates for dead players
+        if (!pSourcePlayer->CanUpdateSync(ucTimeContext) && !pSourcePlayer->IsDead())
```

Comment:
I don't think dead players can be synchronized if the time context doesn't match